import/controls/permissionmodel.cpp

```diff
@@ -103,8 +162,9 @@ void PermissionModel::remove(int index)
 
 void PermissionModel::setCapability(QModelIndex index, int capability)
```

Comment:
Is there any reason not to use `PermissionManager::Capability` here for the second parameter, instead of `int`?

Reply:
I've got something like that
https://bugreports.qt.io/browse/QTBUG-19741
enum doesn't work for Q_INVOKABLE method's in other class than where it is. In some case's I've mistaken.. I will find the places, where it is possible and will fix these